src/logicTemplates/htlc.js

```diff
@@ -98,7 +98,7 @@ function signTransactionWithHTLCUnlock(contract, txn, preImageAsBase64) {
             throw new Error("sha256 hash of preimage did not match stored contract hash")
         }
     } else if (hashFunction == 2) {
-        let actualHashedOutput = keccak256(preImageBytes);
+        let actualHashedOutput = createKeccakHash('keccak256').update(preImageBytes).digest();
```

Comment:
imported `keccak256` but calling `createKeccakHash` this should be working.

Reply:
You're right. Forgot to commit that hunk, I've updated the pull request with the change and rebased it against `develop`.